rfcs/convergence/props-consistency-check.md

```diff
@@ -0,0 +1,391 @@
+# Consistency Check
+
+The following components and their props were reviewed as of 9/7/2021 for consistency.
+
+Accordian, AccordianHeader, AccordianItem, AccordianPanel
+Avatar
+Badge, CounterBadge, PresenceBadge,
+Button, CompoundButton, MenuButton, ToggleButton,
+Divider,
+Image,
+Label,
+Link,Menu, MenuDivider, MenuGroup, MenuGroupHeader,MenuItem, MenuItemCheckbox, MenuItemRadio, MenuList, MenuPopover, MenuTrigger
+Popover, PopoverSurface, PopoverTrigger
+Portal
+
+## Nit: Explicit component type declaration
+
+> Some components declare their type explicity, others do not.
+
+```tsx
+export const Accordion: React.FunctionComponent<AccordionProps & React.RefAttributes<HTMLElement>> = React.forwardRef<
+  HTMLElement,
+  AccordionProps
+>((props, ref) =>
+```
+
+```tsx
+export const Avatar = React.forwardRef((props: AvatarProps, ref: React.Ref<HTMLElement>) =>
+```
+
+## Props declared as interfaces vs types
```

Comment:
from @ecraig12345  in HackMD:
For reference https://github.com/microsoft/fluentui/pull/19376#discussion_r700046363 (also we may have discussed type vs. interface at some point a long time ago but didn't come up with a conclusion)